features/visitor_can_see_products.feature

```diff
@@ -0,0 +1,20 @@
+Feature: Visitor can see products on landing page
+
+    As a visitor,
+    In order to get an overview of the restaurants products
+    I would like to be able to see a menu
+
```

Comment:
is the formatting right under this? Should it all be indented one or two more than the Feature block?

Reply:
Fixed, see latest commit.